File: omninavi/src/main/java/com/omni/omninavi/model/google_navigation/GoogleWaypoint.java
package com.omni.omninavi.model.google_navigation;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 * Created by wiliiamwang on 14/06/2017.
 */

public class GoogleWaypoint implements Serializable {

    @SerializedName("geocoder_status")
    private String geoCoderStatus;
    @SerializedName("place_id")
    private String placeId;
    @SerializedName("types")
    private String[] types;

    public String getGeoCoderStatus() {
        return geoCoderStatus;
    }

    public void setGeoCoderStatus(String geoCoderStatus) {
        this.geoCoderStatus = geoCoderStatus;
    }

    public String getPlaceId() {
        return placeId;
    }

    public void setPlaceId(String placeId) {
        this.placeId = placeId;
    }

    public String[] getTypes() {
        return types;
    }

    public void setTypes(String[] types) {
        this.types = types;
    }
}

File: navisdk/src/main/java/com/omni/navisdk/module/google_navigation/GoogleWaypoint.kt
package com.omni.navisdk.module.google_navigation

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class GoogleWaypoint : Serializable {
    @SerializedName("geocoder_status")
    var geoCoderStatus: String? = null

    @SerializedName("place_id")
    var placeId: String? = null

    @SerializedName("types")
    var types: Array<String>? = null

}